website/docs/docs/dbt-versions/release-notes.md

```diff
@@ -17,7 +17,7 @@ pagination_prev: null
 Release notes are grouped by month for both multi-tenant and virtual private cloud (VPC) environments.
 
 ## December 2025
-
+- **Enhancement**: The dbt VS Code extension now supports registering with OAuth for users in the default region (`US1`) that previously created a dbt account in the past. This makes it easier to register the extension for users who may have forgotten their password or are locked out of their account. For more information, see [Register the extension](/docs/install-dbt-extension#register-the-extension).
```

Comment:
Not sure if I capture this correctly, so lmk what you think:
```suggestion
- **Enhancement**: For users in the default region (`US1`), the dbt VS Code extension now supports registering with OAuth as an alternative to creating a new dbt account. This makes it easier to register the extension for users who may have forgotten their password or are locked out of their account. For more information, see [Register the extension](/docs/install-dbt-extension#register-the-extension).
```

Reply:
great sugg Leona! it's not as an alternative so much so as it replaces having to manually download the dbt_cloud.yml file for users that prev created (but might not remember) a dbt account. i'll rejig it a bit to:

> For users in the default region (`US1`) that previously created a dbt account in the past, the dbt VS Code extension now supports registering with OAuth . This makes it easier to register the extension for users who may have forgotten their password or are locked out of their account. For more information, see [Register the extension](/docs/install-dbt-extension#register-the-extension).